project/eScavenge/CLI.py

```diff
@@ -289,11 +289,39 @@ def answer_question(self, args):
         return "There are no more landmarks!"
     return "Incorrect Answer! The Question Was: \n{}".format(self.game.get_team_landmark(self.current_user).question)
 
+@need_admin
+def edit_penalty_value(self, args):
+    if self.game.started or self.game.ended:
+        return game_currently_running
+    try:
+        penalty_value = self.game.set_point_penalty(args[1])
```

Comment:
args[1] will always be a string, so this should basically always fail. You need to cast it to int here.